packages/client/src/client.ts

```diff
@@ -81,11 +81,13 @@ export async function handleDBStorage({
   url,
   message,
   signer,
+  multicall,
 }: {
   domain: DomainData
   url: string
   message: MessageData
   signer: PrivateKeyAccount
+  multicall?: boolean
```

Comment:
Why do we need a parameter here? My understanding is that a multicall would be treated as another call (setText for example).